Python_projects/github_scraper/github_scrapper.py

```diff
@@ -0,0 +1,90 @@
+import requests
+from bs4 import BeautifulSoup
+import html5lib
+import re
+import csv
+from .constants import *
+
+
+#This function fetches all the github profile links from the search result page and returns list containing the links 
+def get_profile_links(search_query, no_of_results):
+    base_url = BASE_URL
+    github_url = GITHUB_URL
+    links = []
+
+    total_pages = int(no_of_results/10) + 1
+
+    page = 1
+    while page <= total_pages:
+        req = requests.get(base_url + "p=" + str(page) + "&q=" + str(search_query))
```

Comment:
please follow vertical spacing for example , provide one line space between 17 and 18